test/format.dev.spec.js

```diff
@@ -86,7 +86,7 @@ describe('Development format', function() {
     it('should log dates', function() {
       var now = new Date();
       logger.info({}, now);
-      expect(logger._lastTrace).to.be.eql('INFO  ' + now);
+      expect(logger._lastTrace).to.be.eql('INFO  1970-01-01T00:00:00.000Z');
```

Comment:
Because of sinon dark magic at https://github.com/telefonicaid/logops/blob/master/test/environment.js#L20

More Reading at http://sinonjs.org/releases/v1.17.7/fake-timers/